public/static/docs/use-cases/shared-development-server.md

```diff
@@ -37,6 +37,16 @@ to simply move it from an old cache location to the new one:
 $ mv .dvc/cache/* /path/to/dvc-cache
 ```
 
+Now you need to ensure that cache files/directories have appropriate
+permissions, so that they could be accessed by your collegues that are members
+of the same group:
+
+```dvc
+$ sudo find /path/to/dvc-cache -type f -exec chmod 0664 {} \;
+$ sudo find /path/to/dvc-cache -type d -exec chmod 0775 {} \;
+$ sudo chown -R myuser:ourgroup /path/to/dvc-cache/
```

Comment:
Needs clarification that the user needs to use his username and his group, that he shares with his team. Also `sudo` is probably also something that we need to describe better, as it might require asking devops guys or something, not sure if usually DS teams have sudo access on their servers.